src/helpers/PromptsHelper.ts

```diff
@@ -20,7 +20,7 @@ export default class PromptsHelper {
    * @returns boolean
    */
   static isCategorySlidedownConfiguredVersion1(options: SlidedownOptionsVersion1): boolean {
-    return !!options?.categories?.tags && options.categories.tags.length > 0;
+    return (!!options?.categories?.tags.length || 0) > 0;
```

Comment:
`(options?.categories?.tags?.length || 0) > 0;` is what I expect we want here. Is there something this doesn't cover?


My comments on your suggestion of `(!!options?.categories?.tags.length || 0) > 0;`
- `.tags.length` - `tags` could be `null` or `undefined` so shouldn't this be `tags?.length`?
- `!!` would turn a number into a `boolean`. JS allows you to do less than or greater than on booleans but we really should be using the same type.

Reply:
Indeed. That was not deliberate. Good catch.